### PR TITLE
Page element for manage restaurants page

### PR DESCRIPTION
**Pull request by tarasiki**

### CODE REVIEW

**Review by eugenek7 — APPROVED** [review]

File: EasyRestProjectNetTeam2/EasyRestPages/ManageRestaurantsPage.cs [hunk 1]
```diff
@@ -34,87 +38,59 @@ public class ManageRestaurantsPage : BasePage
...
-        public void SendKeysToInputRestaurantAdress(string adress)
+        public ManageRestaurantsPage WaitForArchiveStatus(int timeToWait)
         {
-            _inputRestaurantAdress.SendKeys(adress);
+            WaitVisibilityOfElement(timeToWait, _archivedStatus);
+            return this;
         }
 
-        public void ClickCreateRestaurantButton()
+        public ManageRestaurantsPage WaitForUnarchiveStaus(int timeToWait)
```
Comment (dmatsev) [inline EasyRestProjectNetTeam2/EasyRestPages/ManageRestaurantsPage.cs]: WaitForUnarchiveSta**t**us

File: EasyRestProjectNetTeam2/EasyRestPages/RestaurantsPage.cs [hunk 2]
```diff
@@ -35,8 +34,8 @@ public void WaitAndClickJonsonDetails(int timeToWait)
         public void WaitAndClickJonsonMenu(int timeToWait)
         {
             _johnsonMenu.WaitAndClick(driver, timeToWait);
-        }
-
+        }
```
Comment (dmatsev) [inline EasyRestProjectNetTeam2/EasyRestPages/RestaurantsPage.cs]: you have empty spaces:
image

please do code cleanup

File: EasyRestProjectNetTeam2/EasyRestComponentsObj/ManageResturantComponent.cs [hunk 3]
```diff
@@ -0,0 +1,52 @@
+﻿using EasyRestProjectNetTeam2.EasyRestPages;
+using OpenQA.Selenium;
+using SeleniumExtras.PageObjects;
+
+namespace EasyRestProjectNetTeam2.EasyRestComponentsObj
+{
+    public class ManageResturantComponent:ManageRestaurantsPage
```
Comment (dmatsev) [inline EasyRestProjectNetTeam2/EasyRestComponentsObj/ManageResturantComponent.cs]: why do you use inheritance?

Comment (AhnessP) [inline EasyRestProjectNetTeam2/EasyRestComponentsObj/ManageResturantComponent.cs]: typo in ManageRestaurantComponent

**Review by SerhiiTarasiuk — APPROVED** [review]

**Pull request merged**